pwiz_tools/Skyline/SettingsUI/ViewLibraryDlg.cs

```diff
@@ -695,8 +742,8 @@ public void UpdateUI(bool selectionChanged = true)
 
                 if (-1 != index)
                 {
-                    SpectrumPeaksInfo spectrum;
-                    if (_selectedLibrary.TryLoadSpectrum(_peptides[index].Key, out spectrum))
+                    SpectrumPeaksInfo isLoadableSpectrum;

```

Comment:
Usually, a name like "is..." would be reserved for a boolean value. Maybe something like loadedSpectrum?